contentcuration/contentcuration/static/js/edit_channel/tree_edit/views.js

```diff
@@ -14,28 +14,33 @@ var TreeEditView = BaseViews.BaseView.extend({
 	containers:[],
 	template: require("./hbtemplates/container_area.handlebars"),
 	initialize: function(options) {
-		_.bindAll(this, 'copy_content','delete_content' , 'add_container', 'edit_content', 'toggle_details'/*,'undo_action', 'redo_action'*/);
+		_.bindAll(this, 'copy_content','delete_content' , 'add_container', 'edit_content', 'toggle_details', 'back_to_edit'/*,'undo_action', 'redo_action'*/);
 		this.is_edit_page = options.edit;
 		this.collection = options.collection;
 		this.is_clipboard = options.is_clipboard;
-		this.render();
-		this.queue_view = new QueueView.Queue({
-	 		el: $("#queue-area"),
-	 		collection: this.collection
-	 	});
-	 	$("#queue-area").css("display", (this.is_clipboard || !this.is_edit_page)? "none" : "block");
-	 	$("#main-nav-home-button").removeClass("active");
 
-	 	if(this.is_edit_page){
-			$("#channel-edit-button").addClass("active");
-	 	}else if(!this.is_clipboard){
-	 		$("#channel-preview-button").addClass("active");
-	 	}
+		var self=this;
+		this.display_load("Loading Content...", function(){
+			self.render();
+			self.queue_view = new QueueView.Queue({
+		 		el: $("#queue-area"),
+		 		collection: self.collection
+		 	});
+		 	$("#queue-area").css("display", (self.is_clipboard || !self.is_edit_page)? "none" : "block");
```

Comment:
`.toggle(boolean)` does something similar (but doesn't assume `block`, in case the thing you're showing/hiding is `inline` etc) -- but this is fine for now.